projects/showcase/src/main.ts

```diff
@@ -1,13 +1,7 @@
-import { enableProdMode } from '@angular/core';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { bootstrapApplication } from '@angular/platform-browser';
+import { appConfig } from './app/app.config';
+import { AppComponent } from './app/app.component';
 
-import { AppModule } from './app/app.module';
-import { environment } from './environments/environment';
-
-if (environment.production) {
-  enableProdMode();
-}
-
-platformBrowserDynamic()
-  .bootstrapModule(AppModule)
-  .catch((err) => console.log(err));
+bootstrapApplication(AppComponent, appConfig).catch((err) =>
```

Comment:
Is it possible to inject the directives here instead of per module?

Reply:
Maybe there's a way to define a testing module, add these directives there and import that module here?
See: https://stackoverflow.com/questions/75520858/how-to-use-directive-inside-standalone-component-in-angular